.github/workflows/validation.yaml

```diff
@@ -41,5 +41,6 @@ jobs:
           go vet ./...
       - name: Test
         run: |
-          go test ./... -coverprofile coverage.out -covermode count -tags=unit

```

Comment:
Why are the unit tests removed?

Reply:
All tests are run and nothing is removed. Tests can be run indirectly as well.